### PR TITLE
EPMRPP-47706: Add ui extension loading mechanism

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

**Comment by DiscoElevator** [comment]
Waiting for planned version clarificaion

**Comment by DiscoElevator** [comment]
Will go to 5.2

File: app/src/controllers/plugins/selectors.js [hunk 1]
```diff
@@ -39,12 +39,15 @@ export const pluginsSelector = (state) => domainSelector(state).plugins;
...
   pluginsSelector(state).find((plugin) => plugin.name === name);
 
-const globalIntegrationsSelector = (state) =>
+export const globalIntegrationsSelector = (state) =>
   domainSelector(state).integrations.globalIntegrations || [];
 const projectIntegrationsSelector = (state) =>
   domainSelector(state).integrations.projectIntegrations || [];
 
 export const availablePluginsSelector = createSelector(pluginsSelector, filterAvailablePlugins);
+export const availablePluginNamesSelector = createSelector(availablePluginsSelector, (plugins) =>
```
Comment (AmsterGet) [inline app/src/controllers/plugins/selectors.js]: `availablePluginsSelector` used for project settings page to hide disabled plugins and plugins with admin access only.
So if you want to get all plugin names, please, use `pluginsSelector`.

Comment (DiscoElevator) [reply app/src/controllers/plugins/selectors.js]: done.
Changed to filtering only enabled plugins

**Pull request merged**